src/Copy/BrandCopy.elm

```diff
@@ -542,3 +542,19 @@ brandCopy key =
 
         StoryThree4ImageAlt ->
             CopyText "Illustration of a man looking relaxed and happy in a garden. He is in deep conversation with a woman who is smiling."
+
+
+relatedInfo : Int -> List Int
+relatedInfo storyId =
+    case storyId of
+        1 ->
+            [ 1, 2 ]
+
+        2 ->
+            [ 2, 3 ]
+
+        3 ->
+            [ 3, 4 ]
```

Comment:
As per https://github.com/neontribe/ca-hospice-hope/issues/4 acceptance should be 5,6